stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/DefaultCenterContent.kt

```diff
@@ -0,0 +1,53 @@
+package io.getstream.chat.android.ui.message.composer
+
+import android.content.Context
+import android.text.Editable
+import android.util.AttributeSet
+import android.widget.FrameLayout
+import androidx.core.view.isVisible
+import com.google.android.material.internal.TextWatcherAdapter
+import io.getstream.chat.android.common.state.MessageInputState
+import io.getstream.chat.android.ui.common.extensions.internal.streamThemeInflater
+import io.getstream.chat.android.ui.databinding.StreamUiMessageComposerDefaultCenterContentBinding
+
+internal class DefaultCenterContent : FrameLayout {
```

Comment:
WOuld love documentation on the class and all of its properties and functions. Even if it's internal, it's good to have everything documented with kdocs for sanity